js/audio/interactive/Audio.svelte

```diff
@@ -259,14 +257,14 @@
 						<span class="pinger" />
 						<span class="dot" />
 					</span>
-					Stop recording
+					$_('audio.stop_recording')
```

Comment:
This change seems to have broken the text displayed on the microphone:

<img width="762" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/a52cc35b-8083-4dca-870b-2efe3117cae5">

<img width="796" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/07dc46fa-0df5-40d7-97d8-6886e0ab8c1e">


```py
import gradio as gr

with gr.Blocks() as demo:
    gr.Audio(source="microphone")
    
demo.launch()
```